### DISCUSSION TITLE
First weeks of use

### DISCUSSION THREAD

**Discussion by larskarla** [flobz/psa_car_controller#433] [Ideas]
Lowest trim level Opel Corsa-e, psa_car_controller installed with docker image on Windows 10.

My observations:

I don't get any "Trips" at all, but I get trips in the myopel app with driving time and distance, but unknown start and end adress. Can't find anything in the log about the trips. Therefore, the only thing I can see on the "Summary" tab is the "Average charge speed", everything else is zero.

The "Charge" tab works, lists old charging sessions with start and end time and %, but I miss the remaining time estimated on the page, that number is at least in the log as "remaining_time":390,".
Clicking on the number in the end level column gives an error, "TypeError: strptime() argument 1 must be str, not int"
Charging curves would be nice as well, both for the current and old charging sessions. 

Happy to provide log file or anything else to help the development.

**Comment by Glexu** [comment]
Hi,
i have same issue. I run in docker. Account login was successful. The application does not show me any Trips and Summary, or Map. 
 The only thing that works is the tab with the charge, current mileage and battery status.

Any ideas how to fix it?

**Reply by stevoh6** [reply]
And  do you have records in DB?

**Comment by MartinSax** [comment]
same issue here. I have trips in the my mobile app
(opel corsa-e)
